lib/takeaway.rb

```diff
@@ -0,0 +1,46 @@
+class Takeaway
+  attr_reader :menu, :order, :total
+
+  ERROR = "Total cost does not match the sum of the dishes in your order!"
+
+  def initialize(menu_klass = Menu.new, order_klass = Order.new)
+    @menu = menu_klass
+    @order = order_klass
+  end
+
+  def open_menu
+    menu.open
+  end
+
+  def add_to_order(item, quantity = 1)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)